packages/design-system/src/components/Alert/Alert.test.tsx

```diff
@@ -63,17 +75,21 @@ describe('Alert', function () {
     expect(alert.tabIndex).toBe(-1);
   });
 
+  // experiencing similar issues to analytics hook; is it hooks WC has issues with?
   it('sets tabIndex when alertRef is passed', () => {
     const alertRef = jest.fn();
-    renderAlert({ alertRef });
+    renderAlert({ 'alert-ref': alertRef });
     const alert = screen.getByRole('region');
     expect(alert.tabIndex).toBe(-1);
     expect(alertRef).toHaveBeenCalled();
   });
 
+  // adding native attrs to WC results in multiples being added throughout component
+  // seems like we need to apply native attrs using setAttribute
   it('renders additional attributes', () => {
     const ariaLabel = 'additional aria alert';
-    renderAlert({ 'aria-label': ariaLabel });
```

Comment:
adding additional HTML attrs resulted in strange output; saw multiples of `aria-label` and `arialabel` applied to several nodes, which made it hard for the test to query the correct element to apply the expectation. i _think_ the way to apply native attrs is to use `setAttribute` since this is a native HTML element but i'm not sure. either way, doing that seems to satisfy this test